client/jetpack-cloud/components/sidebar/index.tsx

```diff
@@ -75,14 +74,18 @@ const JetpackCloudSidebar = ( {
 				</SidebarNavigator>
 			</SidebarMain>
 
-			{ ! isJetpackManage && (
+			{ ! isJetpackManage && jetpackAdminUrl && (
 				<SidebarFooter>
-					<SidebarItem
-						label={ translate( 'WP Admin', {
-							comment: 'Jetpack Cloud sidebar navigation item',
-						} ) }
+					<SidebarNavigatorMenuItem
+						title={ translate( 'WP Admin' ) }
 						link={ jetpackAdminUrl }
-						customIcon={ <JetpackIcons icon="wordpress" /> }
+						path={ jetpackAdminUrl }
+						icon={ <JetpackIcons icon="wordpress" /> }
+						onClickMenuItem={ ( link ) => {
+							// TODO: Add track event here.
```

Comment:
Is there any reason why we didn't add the track event from now?

Reply:
I wasn't sure initially what the prefix was for the event name, but I decided to go ahead and use the existing one. Thanks for pointing this out. 👍